### DISCUSSION TITLE
Is mamba-ui fully responsive?

### DISCUSSION THREAD

**Discussion by atoniolo76** [Microwawe/mamba-ui#76] [Q&A] [answered]
Hi,

I am interested in using Mamba UI for my next Svelte project, and I am wondering if mamba is responsive for mobile, as I use DaisyUI and it does not come with responsiveness. Thanks!

**Comment by Microwawe** [comment, marked as answer]
Hi!

Yes all the components and templates should be fully responsive (in case something looks like it isn't, just let me know and I'll try to fix it).

**Comment by vskidevelopers** [comment]
Yes, it is. in case of anything you can always overwrite the tailwind classes.
